### PR TITLE
File Ordered and redundant files removed

### PR DESCRIPTION
**Pull request by Kumar-laxmi**

### CODE REVIEW

**Comment by Kumar-laxmi** [comment]
Now is it ok to merge

File: aboutus.html [hunk 1]
```diff
@@ -74,7 +74,7 @@ <h5 align="center"> Let's have a chat on </h5>
         </p>
 
 
-        <!-- <script src="./JS/about.js"></script> -->
+        <!-- <script src="./JS/about.js"></script> -->   <!-- Why is this commented? -->
```
Comment (flyingsonu122) [inline aboutus.html:77]: Not in use right now

**Review by flyingsonu122 — APPROVED** [review]
Thank you

File: aboutus.html (hunk 1 above)
Comment (Kumar-laxmi) [reply aboutus.html:77]: ok

**Comment by flyingsonu122** [comment]
Add proper links in index.html

**Comment by Kumar-laxmi** [comment]
@flyingsonu122 Please assign this PR the label as ```GSSoC22``` and ```Level2``` or whatever level so that this contribution is been noted in the LeaderBoard of GSSoC

**Pull request merged**